### ISSUE TITLE
Test accepting terms of use with VoiceOver

### ISSUE THREAD

**Issue by AnttiRumpunen** [fmidev/weather-app#1259]
Citizen was unable to accept terms of use and had to ask help from another person.

**Comment by AnttiRumpunen** [comment]
cannot reproduce
